src/NzbDrone.Core/MediaCover/MediaCoverService.cs

```diff
@@ -179,13 +180,22 @@ private void DownloadCover(Artist artist, MediaCover cover, DateTime lastModifie
             }
         }
 
-        //private void DownloadAlbumCover(Album album, MediaCover cover)
-        //{
-        //    var fileName = GetCoverPath(album.ArtistId, cover.CoverType, null, album.Id);
+        private void DownloadAlbumCover(Album album, MediaCover cover, DateTime lastModified)
+        {
+            var fileName = GetCoverPath(album.ArtistId, cover.CoverType, null, album.Id);
```

Comment:
Remove artistId from album cover path, more independent we can make albums the better